src/components/CondoCardModal.vue

```diff
@@ -74,14 +74,33 @@ export default {
       this.$store.commit('setActiveFeature', null);
     },
     addCondoRecords(state) {
-      if(this.$store.state.lastSearchMethod === "geocode") {
+      let mapUnitIds = function(id) {
```

Comment:
Is there a reason you're defining mapUnitIds here as a method inside the `addCondoRecords` method? It seems like it would be cleaner if that were its own method - or simply done inline.

Reply:
If it's not used anywhere else, I tend to just keep functions like these within the method itself. 
It makes it a little easier to sort out when there's a lot going on.